docker-compose.yml

```diff
@@ -22,11 +22,38 @@ services:
       - AWS_ENDPOINT_OVERRIDE=http://localstack:4566/
       - AWS_S3_BUCKET_FAILED_FILES=data-store-failed-files-dev
       - AWS_S3_BUCKET_FIND_DATA_FILES=data-store-find-data
-#      - NOTIFY_FIND_API_KEY=${NOTIFY_FIND_API_KEY:?err}
+      #      - NOTIFY_FIND_API_KEY=${NOTIFY_FIND_API_KEY:?err}
       - FIND_SERVICE_BASE_URL=http://localhost:4002
+      - REDIS_URL=redis://redis-data:6379
     restart: unless-stopped
     depends_on:
       - database
+  celery:
+    build:
+      context: ../funding-service-design-post-award-data-store
+      args:
+        REQUIREMENTS: requirements-dev.txt
+    command: celery -A app.celery_app worker --loglevel INFO
+    volumes: [ '../funding-service-design-post-award-data-store:/app' ]
+    depends_on:
+      - database
+      - redis-data
+    env_file: .env
+    environment:
+      - FLASK_ENV=development
+      - DATABASE_URL=postgresql://postgres:password@database:5432/data_store
+      - AWS_REGION=${AWS_REGION:-eu-central-1}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:-test}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY:-test}
+      - AWS_ENDPOINT_OVERRIDE=http://localstack:4566/
+      - AWS_S3_BUCKET_FAILED_FILES=data-store-failed-files-dev
+      - AWS_S3_BUCKET_FIND_DATA_FILES=data-store-find-data
```

Comment:
Does the worker actually need these env vars? Does it start up without them? Would be good to get the env vars down to a minimal set.

Reply:
Removed these from celery setup.